examples/anthos-service-mesh-multicluster/infrastructure/network.tf

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Creating the network
+# Create two VPCs, each with a private GKE cluster and one bastion server
+
+# VPC 3
+/**
+resource "google_compute_network" "asm-vpc-3" {
```

Comment:
Why don't we create the VPC via Terraform?

Reply:
Done!